pkg/cmd/create/cluster/scenario/create/values-template.yaml

```diff
@@ -31,10 +32,20 @@ managedCluster:
     awsAccessKeyID:
     awsSecretAccessKeyID:
     region: # Region (ie: us-east-1)
-    master:
-      type: m5.xlarge
-    worker:
-      type: m5.xlarge
+    #Optional
```

Comment:
Wonder if the user will have trouble getting the yaml indentation correct when they remove the # ?   Not sure if there is a better way to show this...

Reply:
I think is ok